python/neutron-understack/neutron_understack/config.py

```diff
@@ -4,30 +4,8 @@
     cfg.StrOpt(
         "provisioning_network",
         help="provisioning_network ID as configured in ironic.conf",
+        default="change_me",
     ),
-    cfg.StrOpt(
-        "argo_workflow_sa",
-        default="workflow",
-        help="ServiceAccount to submit Workflow as",
-    ),
-    cfg.StrOpt(
-        "argo_api_url",
-        default="https://argo-server.argo.svc.cluster.local:2746",
-        help="URL of the Argo Server API",
-    ),
-    cfg.StrOpt(
-        "argo_namespace",
-        default="argo-events",
-        help="Namespace to submit the Workflows to",
-    ),
-    cfg.IntOpt(
-        "argo_max_attempts",
-        default=15,
-        help="Number of tries to retrieve the Workflow run result. "
-        "Sleeps 5 seconds between attempts.",
-    ),
-    cfg.BoolOpt("argo_dry_run", default=True, help="Call Undersync with dry-run mode"),
-    cfg.BoolOpt("argo_force", default=False, help="Call Undersync with force mode"),
```

Comment:
I added these options to the undersync API because they might be useful for troubleshooting, etc., but I don't think they need to be exposed here unless we have a specific need for them.  I can maybe see a reason to have dry_run but certainly "force" would not be a good thing to have permanently switched on.

Reply:
implemented